tensorzero-core/src/db/clickhouse/mod.rs

```diff
@@ -301,6 +301,18 @@ impl ClickHouseConnectionInfo {
     }
 }
 
+impl Display for ClickHouseConnectionInfo {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self.client_type() {
+            ClickHouseClientType::Production => {
+                write!(f, "enabled (database = {})", self.database())
+            }
+            ClickHouseClientType::Mock => write!(f, "mocked"),
```

Comment:
Can we actually keep calling this "fake"? Fake vs. Mock have specific connotations (Fake = faked implementation that does real things like the in-memory implementation; mock = a shim that we can directly specify input/output matching in tests). Here the fake shouldn't even be included in production code so maybe we add #cfg_attr(test) to all of it?

Reply:
@shuyangli renamed back. as for `test`-only, defer to a future PR as that might affect `e2e_tests` (afaik `cargo run-e2e` is not `test`)